src/encoded/submit.py

```diff
@@ -0,0 +1,3165 @@
+            "diagnosis age of onset units": "diagnosis_onset_age_units",
+        },
+        "family": {},
+        "sample": {
+            "date collected": "specimen_collection_date",
+            "location stored": "specimen_storage_location",
+            "specimen id": "specimen_accession",
+            "transport method": "transported_by",
+            "sequencing ref lab": "sequencing_lab",
+            "date rec'd at ref lab": "date_received",
+            "specimen accepted by ref lab": "specimen_accepted",
+            "sample id by ref lab": "sequence_id",
+            "req type": "requisition_type",
+            "date req rec'd": "date_requisition_received",
+            "physician/provider": "ordering_physician",
+            "test requested": "workup_type",
+        },
+        "requisition": {
+            "req accepted y/n": "accepted_rejected",
+            "reason rejected": "rejection_reason",
+            "corrective action taken": "corrective_action",
+            "corrective action taken by": "action_taken_by",
+            "correction notes": "notes",
+        },
+    }
+)
+
+
+ABBREVS = {
+    "male": "M",
+    "female": "F",
+    "unknown": "U",
+    "yes": "Y",
+    "no": "N",
+    "p": "proband",
+    "mth": "mother",
+    "fth": "father",
+    "sf": "sibling",
+}
+YES_VALS = ["y", "yes"]
+
+# SS at end refers to spreadsheet, to distinguish from prop names in schema if we need
+# vars for those at any point.
+SS_INDIVIDUAL_ID = "individual id"
+SS_FAMILY_ID = "family id"
+SS_SEX = "sex"
+SS_SPECIMEN_ID = "specimen id"
+SS_ANALYSIS_ID = "analysis id"
+SS_RELATION = "relation to proband"
+SS_REPORT_REQUIRED = "report required"
+SS_PROBAND = "proband"
+
+REQUIRED_COLS_FOR_CASE = [SS_ANALYSIS_ID, SS_SPECIMEN_ID]
+REQUIRED_COLS_FOR_ACCESSIONING = REQUIRED_COLS_FOR_CASE + [
+    SS_INDIVIDUAL_ID,
+    SS_SEX,
+    SS_RELATION,
+    SS_REPORT_REQUIRED,
+]
+REQUIRED_COLS_FOR_PEDIGREE = [SS_FAMILY_ID, SS_INDIVIDUAL_ID, SS_SEX, SS_PROBAND]
+
+# half-siblings not currently supported, because pedigree info is needed to know
+# which parent is shared. Can come back to this after pedigree processing is integrated.
+SIBLING_LABEL = "sibling"
+SIBLINGS = [
+    "sibling",
+    "brother",
+    "sister",
+    "full sibling",
+    "full brother",
+    "full sister",
+]
+
+RELATIONS = SIBLINGS + ["proband", "mother", "father"]
+
+POST_ORDER = [
+    "file_submitted",
+    "sample",
+    "individual",
+    "family",
+    "sample_processing",
+    "report",
+    "case",
+]
+
+
+LINKTO_FIELDS = [  # linkTo properties that we will want to patch in second-round
+    "samples",
+    "members",
+    "mother",
+    "father",
+    "proband",
+    "report",
+    "individual",
+    "sample_processing",
+    "families",
+    "family",
+    "files",
+    "related_files",
+]
+
+
+ID_SOURCES = ["UDN"]
+
+HPO_TERM_ID_PATTERN = re.compile(r"^HP:[0-9]{7}$")
+MONDO_TERM_ID_PATTERN = re.compile(r"^MONDO:[0-9]{7}$")
+
+
+def submit_metadata_bundle(
```

Comment:
I think everything in this file, except for dependencies of this function (and maybe others I've missed) is specific to CGAP and can be removed.

Reply:
Actually there is no reference anywhere to submit_metadata_bundle - the that an indication that something is off? It is referenced in cgap-portal in `ingestion.processors.handle_metadata_bundle`.